### PR TITLE
Parse KML Placemark.visibility

### PR DESCRIPTION
**Pull request by fabriziomoscon**
Return the visibility children of KML Placemarkers inside the feature properties

### CODE REVIEW

File: togeojson.js [hunk 1]
```diff
@@ -238,6 +239,9 @@ var toGeoJSON = (function() {
                         properties[simpleDatas[i].getAttribute('name')] = nodeVal(simpleDatas[i]);
                     }
                 }
+                if (typeof visibility !== undefined && visibility != null) {
```
Comment (tmcw) [inline togeojson.js]: the typeof is not necessary here, you can simply ask `if (visibility)`.

typeof tests are necessary if there's a chance the variable has not been declared, and asking its value would trigger a ReferenceError. In this code, you can be confident that the variable is defined.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (tmcw) [inline .gitignore]: Please remove this: node_modules should be ignored in your global .gitignore

**Comment by fabriziomoscon** [comment]
@tmcw I have pushed the changes you requested

**Comment by tmcw** [comment]
:+1: awesome, thanks!

**Pull request merged**